.travis.yml

```diff
@@ -26,12 +26,22 @@ env:
         - DMD=2.071.2.s1 DIST=xenial F=production
         - DMD=2.071.2.s1 DIST=xenial F=devel
 
+# Don't build tags already converted to D2
+if: NOT tag =~ ^.*\+d2^
```

Comment:
To continue my question from beaver PR - I really think this check is much better in actual convert script, less chance to forget this line in travis script and get things broken.

Reply:
Mmm, for me it's better in here, as we don't even want to trigger a travis build at all for a tag like this (like the status shouldn't be set, if we run the job anyway and make it a NOP it will get a bogus status).

Maybe we can include the check in both parts, but then we might forget to keep them in sync.

Reply:
OTOH, I'm failing and getting these conditions to work. In my previous tests, using `if: tag IS present` in the stage works fine (you can see in this PR job that there is no `D2 Release` stage), but in here the job is still triggered for the `+d2` tag. I don't know if I'm getting the regex wrong or what, I will contact travis support.

Reply:
Fixed, I was just mixing up `^` and `$` :roll_eyes: